release-team/role-handbooks/docs/README.md

```diff
@@ -746,7 +746,8 @@ Announce that `[future release]` branch is open for new feature docs on slack #s
 
 - Clean Up
 ⚠️  Assign any issues / feature / PRs (that missed the release) to the new Docs Lead
-⚠️  Create a PR to remove shadows from [milestone maintainers](https://github.com/orgs/kubernetes/teams/kubernetes-milestone-maintainers) 
+⚠️  Create a PR to remove lead and shadows from [milestone maintainers](https://github.com/orgs/kubernetes/teams/website-milestone-maintainers/)
+⚠️  Create a PR to remove lead from [sig-docs-en-owners](https://github.com/orgs/kubernetes/teams/sig-docs-en-owners)
```

Comment:
We tend to leave the docs lead in `sig-docs-en-owners` so that there is a pool of PR wranglers are available to help out. I brought this up and vaguely remember I was told not to remove. cc: @jimangel for clarification